recipes/multiqc/meta.yaml

```diff
@@ -60,7 +60,6 @@ test:
     - multiqc.modules.busco
     - multiqc.modules.bustools
     - multiqc.modules.ccs
-    - multiqc.modules.checkqc
```

Comment:
Did you mean to remove these? I guess not?

Reply:
No, now copied the checks from MQC 1.12 release to get this right again. Hopefully finding the problem soon now